test/jdk/jdk/internal/vm/Continuation/Basic.java

```diff
@@ -37,7 +37,18 @@
 * @run testng/othervm --enable-preview -Xcomp -XX:-TieredCompilation -XX:CompileOnly=jdk/internal/vm/Continuation,Basic -XX:CompileCommand=exclude,Basic.manyArgsDriver Basic
 * @run testng/othervm --enable-preview -Xcomp -XX:-TieredCompilation -XX:CompileOnly=jdk/internal/vm/Continuation,Basic -XX:CompileCommand=exclude,jdk/internal/vm/Continuation.enter Basic
 * @run testng/othervm --enable-preview -Xcomp -XX:-TieredCompilation -XX:CompileOnly=jdk/internal/vm/Continuation,Basic -XX:CompileCommand=inline,jdk/internal/vm/Continuation.run Basic
+*/
+
+/**
+* @test
+* @requires vm.continuations
+* @requires vm.debug
+* @modules java.base/jdk.internal.vm
+* @build java.base/java.lang.StackWalkerHelper
 *
+* @run testng/othervm --enable-preview -XX:+VerifyStack -Xint Basic
+* @run testng/othervm --enable-preview -XX:+VerifyStack -Xcomp -XX:TieredStopAtLevel=3 -XX:CompileOnly=jdk/internal/vm/Continuation,Basic Basic
```

Comment:
Why 3?  Isn't C1 only equivalent to -XX:TieredStopAtLevel=1 ?

Reply:
It's C1 with profiling. Shouldn't matter here.